modules/hcp-kubeconf-nodepool.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-machine-config.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-kubeconf-nodepool_{context}"]
+= Referencing the kubelet configuration in node pools
+
+To reference your kubelet configuration in node pools, you add the kubelet configuration in a config map and then apply the config map in the `NodePool` resource.
+
+.Procedure
+
+. Add the kubelet configuration inside of a config map in the management cluster by entering the following information:
++
+.Example `ConfigMap` object with the kubelet configuration
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: <configmap_name> <1>
+  namespace: clusters
+data:
+  config: |
+    apiVersion: machineconfiguration.openshift.io/v1
+    kind: KubeletConfig
+    metadata:
+      name: <kubeletconfig_name> <2>
+    spec:
+      kubeletConfig:
+        registerWithTaints:
+        - key: "karpenter.sh/unregistered"
```

Comment:
let's dont use karpentetr as an example here as it can be confusing

Reply:
Changed. 👍